src/server/game/Battlefield/Battlefield.h

```diff
@@ -101,321 +206,87 @@ class TC_GAME_API BfCapturePoint
+        float _value;
+        TeamId _team;
 
         // Objective states
-        BattlefieldObjectiveStates m_OldState;
-        BattlefieldObjectiveStates m_State;
+        BattlefieldObjectiveStates _oldState;
+        BattlefieldObjectiveStates _state;
 
         // Neutral value on capture bar
-        uint32 m_neutralValuePct;
-
-        // Pointer to the Battlefield this objective belongs to
-        Battlefield* m_Bf;
+        uint32 _neutralValuePct;
 
         // Capture point entry
-        uint32 m_capturePointEntry;
+        uint32 _capturePointEntry;
 
         // Gameobject related to that capture point
-        ObjectGuid m_capturePointGUID;
+        ObjectGuid _capturePointGUID;
+
+        // Pointer to the Battlefield this objective belongs to
+        Battlefield* _battlefield;
 };
 
-class TC_GAME_API BfGraveyard
+class TC_GAME_API BattlefieldGraveyard
 {
     public:
-        BfGraveyard(Battlefield* Bf);
+        BattlefieldGraveyard(Battlefield* battlefield);
+        void Initialize(TeamId initialTeam, uint32 id);
 
-        // Method to changing who controls the graveyard
-        void GiveControlTo(TeamId team);
-        TeamId GetControlTeamId() const { return m_ControlTeam; }
+        // Set spirit creature for the graveyard
+        void SetSpirit(Creature* creature, TeamId team);
 
         // Find the nearest graveyard to a player
-        float GetDistance(Player* player);
-
-        // Initialize the graveyard
-        void Initialize(TeamId startcontrol, uint32 gy);
-
-        // Set spirit service for the graveyard
-        void SetSpirit(Creature* spirit, TeamId team);
+        float GetDistance(Player* player) const;
 
         // Add a player to the graveyard
-        void AddPlayer(ObjectGuid player_guid);
+        void AddPlayer(ObjectGuid playerGUID);
 
         // Remove a player from the graveyard
-        void RemovePlayer(ObjectGuid player_guid);
+        void RemovePlayer(ObjectGuid playerGUID);
 
         // Resurrect players
         void Resurrect();
 
+        // Method to changing who controls the graveyard
+        void GiveControlTo(TeamId team);
+
         // Move players waiting to that graveyard on the nearest one
         void RelocateDeadPlayers();
 
         // Check if this graveyard has a spirit guide
-        bool HasNpc(ObjectGuid guid);
+        bool HasCreature(ObjectGuid guid) const;
 
-        // Check if a player is in this graveyard's resurrect queue
-        bool HasPlayer(ObjectGuid guid) { return m_ResurrectQueue.find(guid) != m_ResurrectQueue.end(); }
+        TeamId GetSpiritTeamId(ObjectGuid guid) const;
 
-        // Get the graveyard's ID.
-        uint32 GetGraveyardId() const { return m_GraveyardId; }
-
-    protected:
-        TeamId m_ControlTeam;
-        uint32 m_GraveyardId;
-        ObjectGuid m_SpiritGuide[BG_TEAMS_COUNT];
-        GuidSet m_ResurrectQueue;
-        Battlefield* m_Bf;
-};
-
-class TC_GAME_API Battlefield : public ZoneScript
```

Comment:
u what? I've been updating every old and wrong comment and every log
the moment I start updating battlefield class, is when I'll put the doc